advanced_source/visualizing_gradients_tutorial.py

```diff
@@ -0,0 +1,426 @@
+"""
+Visualizing Gradients
+=====================
+
+**Author**: `Justin Silver <https://github.com/j-silv>`_
+
+By performance and efficiency reasons, PyTorch does not save the
```

Comment:
Don't think we need any reasoning here: "By performance and efficiency reasons,"  these gradients aren't saved because we don't need them.